src/shared/actions/predicatesThunks.ts

```diff
@@ -103,11 +103,12 @@ export const executePreviewQuery = (query: string) => async (
     // which means we have to drag around all this asAssignment
     // garbage to be able to run a query instead of just being able
     // to executeQuery as normal
-    const variableAssignments = getVariables(state)
+    const allVariables = getVariables(state)
+    const variableAssignments = allVariables
       .map(v => asAssignment(v))
       .filter(v => !!v)
     const windowVars = getWindowVars(query, variableAssignments)
-    const extern = buildVarsOption([...variableAssignments, ...windowVars])
+    const extern = buildUsedVarsOption(query, allVariables, windowVars)
```

Comment:
good house keeping here, but this thunk file isn't being used. We should see if we can remove this file (and everything associated with it) so that other people don't maintain some code that hasn't made traction in over a year

Reply:
it's not, trust me

Reply:
Ok will remove it then

Reply:
Don’t worry about it. It’s out of scope. I’ll take care of it